docs/weaver_code_anatomy.md

```diff
@@ -62,25 +59,35 @@ code: |
   if not defined("interview_metadata['a_258e_motion_for_impoundment']"):
     interview_metadata.initializeObject('a_258e_motion_for_impoundment')
   interview_metadata['a_258e_motion_for_impoundment'].update({
-    'title': '209a 258e motion for impoundment',
-    'short title': '209a 258e motion for impoundment',
-    'description': '209a 258e motion for impoundment',
-    'original_form': '',
-    'allowed courts': [
-      'Land Court',
+    "al_weaver_version": "0.82",
+    "generated on": "2021-05-28",
+    "title": "209a 258e motion for impoundment",
+    "short title": "209a 258e motion for impoundment",
+    "description": "209a 258e motion for impoundment",
+    "original_form": "",
+    "allowed courts": [
+      "Land Court",
     ],
-    'categories': [
-      'Guardianship',
+    "categories": [
+      "BE-04-00-00-00",
```

Comment:
Why do we have our own place for these LIST tags? Docassemble seems to already have a place for them in metadata. Maybe a question for a different time, though.

Reply:
:shrug:   Definitely out of scope for this PR

Reply:
Also, I think it's so we can still view the categories of combined interviews. Not sure if there's a way to keep information from multiple metadata blocks from different interviews without this block.